haystack/agents/utils.py

```diff
@@ -2,7 +2,7 @@
 
 from haystack.agents.types import Color
 
-STREAMING_CAPABLE_MODELS = ["davinci"]
+STREAMING_CAPABLE_MODELS = ["davinci", "gpt-3.5"]
```

Comment:
I think we can add gpt-4 as well

Reply:
Yeah, soon after Claude and Cohere command. They all have streaming capabilities